frontend/src/todo/components/task-list/task-list.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { render, screen, fireEvent } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import "@testing-library/jest-dom"
+
+import { TaskList } from "./task-list"
+
+describe("TaskList Component", () => {
+  describe("컴포넌트 랜더링됐을때 화면", () => {
+    it("text들이 화면에 랜더링 나오는지 확인", () => {
+      const data = [
+        { id: 1, title: "hellllo", completed: false, deleteText: () => console.log("helllo") },
+        { id: 2, title: "점심 식사 구하기", completed: true, deleteText: () => console.log("hihihi") },
+        { id: 3, title: "공부 계획 세우기", completed: false, deleteText: () => console.log("third text") },
+      ]
+      render(<TaskList userData={data} />)
+
+      const firstPropsText = screen.getByText("점심 식사 구하기")
+      const deleteButtons = screen.getAllByRole("button")
+
+      expect(firstPropsText).toBeInTheDocument()
+      expect(deleteButtons).toHaveLength(3)
```

Comment:
수고하셨습니다.
작성하신 방법도 맞지만, 두가지 모두 이미 이전 테스트에서 확인한 내용이니 아래 코드로도 충분하다고 생각합니다. 예시입니다.

```suggestion
      const tasks = screen.getAllByRole("listitem")      
      expect(tasks).toHaveLength(3)
```